ui/packages/consul-ui/app/services/repository/permission.js

```diff
@@ -148,6 +148,16 @@ export default class PermissionService extends RepositoryService {
   async findAll(params) {
     params.resources = REQUIRED_PERMISSIONS;
     this.permissions = await this.findByPermissions(params);
+    /**/
+    // Temporarily revert to pre-1.10 UI functionality by overwriting frontend
+    // permissions. These are used to hide certain UI elements, but they are
+    // still enforced on the backend.
```

Comment:
Appreciate the comment for context. I know that the first word in the comment is "Temporarily," but I wonder if we should be a little more explicit about the fact that we intend to return to the intended behavior once some back-end dependencies have been resolved? We don't need to commit to a specific time frame but I feel like making the intentions clear could be beneficial to external contributors.

Reply:
We could mention GH-11098 specifically as the issue to follow for a longer-term fix.

Reply:
Added some more detail here inc. GH issue link

Reply:
:shipit: